.github/scripts/deploy.sh

```diff
@@ -3,15 +3,15 @@
 version=$(npm pkg get version | sed 's/"//g')
 bucket=$1
 
-aws s3api put-object --bucket $bucket --key "content/$version/cwr.js" --body build/assets/cwr.js
-aws s3api put-object --bucket $bucket --key "content/$version/LICENSE-THIRD-PARTY" --body LICENSE-THIRD-PARTY
+aws s3api put-object --bucket $bucket --key "content/$version/cwr.js" --body build/assets/cwr.js --cache-control max-age=604800
```

Comment:
Does it make sense to set these even longer, like 1 year?

Reply:
I'm not sure. It's about risk -- what is the likelihood that a specific version will need to be modified?

In theory a version should never change once deployed, but my preference is still to reduce risk, at least for now, by limiting a potential impact to one week instead of one year.